main.go

```diff
@@ -9,13 +9,16 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	resizeController "github.com/kubernetes-csi/external-resizer/pkg/controller"
```

Comment:
I don't have much context but I can share that the external-resizer runs as a sidecar next to a CSI driver. It is one of the many CSI sidecars - external-provisioner, external-attacher, csi-driver-node-registrar and liveness-probe. Isn't possible to run the external-resizer as a sidecar container?

Reply:
You are right @ialidzhikov, the resizer would fit better as a separate container. Such change though would require a change in the directory structure. I can draft a solution as such, but I would like a comment also from the maintainers if they are okay with such a change. What do you think @derekbit ?